lib/mayaUsd/fileio/shading/shadingModeExporterContext.cpp

```diff
@@ -357,9 +357,8 @@ UsdMayaShadingModeExportContext::GetAssignments() const
     return ret;
 }
 
-static UsdPrim _GetMaterialParent(
+static SdfPath _GetCommonAncestor(
```

Comment:
Factored out the code to find the common ancestor into its own function for clarity.